backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -134,22 +135,19 @@ export class UserResolver {
     email = email.trim().toLowerCase()
     const dbUser = await findUserByEmail(email)
     if (dbUser.deletedAt) {
-      logger.error('The User was permanently deleted in database.')
-      throw new Error('This user was permanently deleted. Contact support for questions.')
+      throw new LogError('This user was permanently deleted. Contact support for questions', dbUser)
     }
     if (!dbUser.emailContact.emailChecked) {
-      logger.error('The Users email is not validate yet.')
-      throw new Error('User email not validated')
+      throw new LogError('The Users email is not validate yet', dbUser)
     }
+    // TODO: at least in test this does not work since `dbUser.password = 0` and `BigInto(0) = 0n`
```

Comment:
Is this even possible? Do we have data where `dbUser.emailContact.emailChecked` and the user has no passwaord?

Reply:
This is to clarify, hence the todo. I think we extend the scope of this PR by changing the logic of our login mechanism.